generate/testdata/snapshots/TestGenerateErrors-InvalidSchema-go

```diff
@@ -1 +1 @@
-testdata/errors/InvalidSchema.schema.graphql:4: invalid schema file testdata/errors/InvalidSchema.schema.graphql: Expected :, found }
+testdata/errors/InvalidSchema.schema.graphql:4: invalid schema: Expected :, found }
```

Comment:
Finding the exact issue will be tough just by seeing the new error message. Previous message was better IMO.

Reply:
I think the filename is still there, just in a different place:
```
 testdata/errors/InvalidSchema.schema.graphql:4: invalid schema: Expected :, found } 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```